### ISSUE TITLE
Add Frame to Allow Cursor Support in Fullscreen

### ISSUE THREAD

**Issue by jodylecompte** [poush/H2#73]
A frame needs to be added to facilitate being able to control the fullscreen window by means of a mouse as well. For example, in Chrome if your window is fullscreeened you are still able to access a button by mousing over the top region of the screen to access an Exit button.

@poush reported that the frame was not working as intended on MacOS and I reproduced the same on Windows 10 and Ubuntu 18.04.

**Comment by poush** [comment]
To reproduce:
1. Start H2 from master branch
2. Press `Alt+Shift+f` for full screen
3. Now the top menu doesn't show. The frame controls are not showing up. All you can do this change the screen or press Esc.

**Comment by iSasFTW** [comment]
@poush Could this be something like what some browsers (like Chrome, for example) do, where when you mouse over the top of the screen, an exit fullscreen button shows up?

**Comment by poush** [comment]
Yes. In macOS, you can see those buttons when you hover at top left but we have to work
